src/plugins/augeas/CMakeLists.txt

```diff
@@ -28,6 +28,11 @@ if (DEPENDENCY_PHASE)
 	set (AUGEAS_PLUGIN_LIBRARIES ${LIBAUGEAS_LIBRARIES} ${LIBXML2_LIBRARIES})
 endif ()
 
+# ——————————————————————————————————————————————————————————
+# TODO: Remove this after we enabled `-Wconversion` globally
+set_source_files_properties (aug.h augeas.c PROPERTIES COMPILE_FLAGS "-Wconversion")
```

Comment:
Can this also be set for the other plugins or maybe even globally already?

Reply:
I already tried that, but in my opinion it was just too much work to do it at once. That is why I decided to introduce the compile flag in multiple smaller pull requests.